scripts/globals/abilities/pets/mountain_buster.lua

```diff
@@ -23,5 +23,10 @@ function onPetAbility(target, pet, skill)
 	target:delHP(totaldamage);
 	target:updateEnmityFromDamage(pet,totaldamage);
 
+	if (damage > 0) then
+	    if (target:hasStatusEffect(EFFECT_BIND) == false) then
+	    target:addStatusEffect(EFFECT_BIND, 1, 0, duration);

```

Comment:
@AaronDemoncia After fixing that nil value Hozu pointed out, please squash the commits:

(In the future you can use the "amend" and "force push" features to avoid making extra commits when changing things, if you prefer).